graphics/lvgl/Makefile

```diff
@@ -106,12 +107,14 @@ $(LVGL_UNPACKNAME): $(LVGL_TARBALL)
 	$(Q) touch $(LVGL_UNPACKNAME)
 
 # Download and unpack tarball if no git repo found
+
 ifeq ($(wildcard $(LVGL_UNPACKNAME)/.git),)
```

Comment:
The LVGL application requires #include <lvgl/lvgl.h> to compile
if the lvgl source is already built "lvgl.h" will be present in (LVGL_UNPACKNAME)/lvgl.h and it will return true.
[2] without Indent I see makefile errors.

Reply:
> The LVGL application requires #include <lvgl/lvgl.h> to compile if the lvgl source is already built "lvgl.h" will be present in (LVGL_UNPACKNAME)/lvgl.h and it will return true. 

> [2] without Indent I see makefile errors.

But the ci run many boards which enable lvgl, no error report before this patch.

Reply:
Okay I will check on that in my local machine on if the indent is required or not, thanks.

Reply:
> I just try sim:lvgl_fb, but can't reproduce your symptom:
> <img alt="image" width="389" src="https://user-images.githubusercontent.com/18066964/218812978-25496046-adfb-49ba-8d78-b28609cc7dec.png">
> lvgl package only download the first time.

Maybe a macOS issue, will try another fix then.

Reply:
Ok, let's wait your finding.